examples/api-demo-1-list-projects.py

```diff
@@ -12,7 +13,8 @@ def parse_command_line_args():
     return parser.parse_args()
 
 
-snyk_token = get_token("snyk-api-token")
+snyk_token_path = get_default_token_path()
+snyk_token = get_token(snyk_token_path)
```

Comment:
I'd probably wrap this into a single function. The path isn't used anywhere else and can be just an implementation of a higher-level function, eg `get_snyk_token()`

Reply:
I separated it like this to improve testability.

Reply:
That doesn't need to bleed over to the user-facing scripts though. A good rule of thumb is to [test the interface rather than the implementation](https://www.richardlord.net/blog/tdd/test-the-interface-not-the-implementation.html). Over testing the implementation tends to result in tightly coupled tests which make refactoring more costly.

Happy for you to merge, as not a big deal. But worth noting.